### DISCUSSION TITLE
mosh

### DISCUSSION THREAD

**Discussion by giscus** [AMD-NICK/blog.amd-nick.me#20] [Blog comments]
# mosh

Обычный SSH работает по TCP. Из-за этого при потери соединения он сразу дисконнектится и нужно переподключаться заново, теряя всю сессию, если только не используется Screen или Tmux.

https://blog.amd-nick.me/mosh

**Comment by TheSilverDreams** [comment]
Виндой пользуюсь, дело привычки, жаль mosh не поддерживаеться
